### PR TITLE
Adding docgen

### PR DESCRIPTION
**Pull request by sullof**
Using OpenZeppelin `solidity-codegen` to add documentation in `docs`.

### CODE REVIEW

File: contracts/erc/IERC6454.sol [hunk 1]
```diff
@@ -1,23 +1,23 @@
 // SPDX-License-Identifier: MIT
 
 /// @title EIP-6454 Minimalistic Non-Transferable interface for NFTs
```
Comment (andyscraven) [inline contracts/erc/IERC6454.sol]: Why is this using /// instead of the /* */ format?

File: contracts/manager/Actor.sol [hunk 2]
```diff
@@ -6,43 +6,43 @@ pragma solidity ^0.8.20;
 // import {console} from "hardhat/console.sol";
```
Comment (andyscraven) [inline contracts/manager/Actor.sol]: Needs removing really before the audit.

File: contracts/manager/CrunaManager.sol [hunk 3]
```diff
@@ -379,6 +379,9 @@ contract CrunaManager is Actor, CrunaManagerBase {
     return CrunaPluginBase(_pluginAddress(nameId_, salt));
   }
 
+  /// @dev returns the address of a deployed plugin
```
Comment (andyscraven) [inline contracts/manager/CrunaManager.sol]: Same format issue of /// not sure why.

File: contracts/manager/CrunaManager.sol [hunk 4]
```diff
@@ -414,18 +426,30 @@ contract CrunaManager is Actor, CrunaManagerBase {
     return (active, disabled);
   }
 
+  /// @dev Internal function to disable a plugin but index and key
```
Comment (andyscraven) [inline contracts/manager/CrunaManager.sol]: The last comment on this issue as there seems to be quite a lot, so there must be a reason.

**Review by andyscraven — APPROVED** [review]

File: contracts/manager/Actor.sol (hunk 2 above)
Comment (sullof) [reply contracts/manager/Actor.sol]: It is not needed, I think; it is commented. But it is quite easy to remove all of them.

**Pull request merged**